modules/administration_manual/pages/configuration/files/encryption/disabling-encryption.adoc

```diff
@@ -0,0 +1,28 @@
+[[disabling-encryption-1]]
+= Disabling Encryption
+
+You may disable encryption only with `occ`. Make sure you have backups
+of all the encryption keys, including those for all users. When you do,
+put your ownCloud server into single-user mode, and then disable your
+encryption module with this command:
+
+....
+sudo -u www-data php occ maintenance:singleuser --on
+sudo -u www-data php occ encryption:disable
+....
+
+IMPORTANT: Encryption cannot be disabled without the user’s password or xref:how-to-enable-users-file-recovery-keys[file recovery key]. If you don’t have access to at least one of these then there is no way to decrypt all files.
+
+Then, take it out of single-user mode when you are finished with this
+command:
+
+....
+sudo -u www-data php occ maintenance:singleuser --off
+....
+
+It is possible to disable encryption with the file recovery key, _if_ every user uses them.
+If so, "decrypt all" will use it to decrypt all files.
+
+NOTE: It is *not* planned to move this to the next user login or a background job. If that was done, then login passwords would need to be stored in the database, which could be a security issue.
```

Comment:
``It is *not* planned to move this`` --> what means ``this`` ?
``If that was done`` --> what was done ?

Reply:
I think this was existing text that updated the formatting of. I'll revise it and reword. Thanks for pointing it out.